src/Pdf.php

```diff
@@ -262,6 +262,16 @@ public function getPdfFilename()
         return $this->_tmpPdfFile->getFileName();
     }
 
+    /**
+     * Returns an options
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return $this->_options;
+    }
```

Comment:
I'm not sure, if I want to expose this. Can you give a use case where someone would ever need the content of the internal `_options` array?

Reply:
for testing purposes - it's definitely valid